src/lug/System/Exception.cpp

```diff
@@ -0,0 +1,72 @@
+#include "../../../include/lug/System/Exception.hpp"
```

Comment:
You just need to write `#include <lug/System/Exception.hpp>`
If you don't do it like that it won't work on the user side. Maybe try to compile the library and test it like an user?